File: app/src/main/java/com/kieran/winnipegbusbackend/ServiceAdvisories/AffectedStop.java
package com.kieran.winnipegbusbackend.ServiceAdvisories;

import java.io.Serializable;

public class AffectedStop implements Serializable {
    private String affectedStop;
    private String affectedRoutes;

    public AffectedStop(String affectedStop, String affectedRoutes) {
        this.affectedStop = affectedStop;
        this.affectedRoutes = affectedRoutes;
    }

    public String getAffectedStop() {
        return affectedStop;
    }

    public String getAffectedRoutes() {
        return affectedRoutes;
    }

    @Override
    public String toString() {
        return affectedStop + " " + affectedRoutes + "\n";
    }
}

File: app/src/main/java/com/kieran/winnipegbusbackend/ServiceAdvisories/AffectedStop.kt
package com.kieran.winnipegbusbackend.ServiceAdvisories

import java.io.Serializable

class AffectedStop(val affectedStop: String, val affectedRoutes: String) : Serializable {

    override fun toString(): String {
        return affectedStop + " " + affectedRoutes + "\n"
    }
}
